xbmc/utils/GUIUtils.h

```diff
@@ -0,0 +1,45 @@
+/*
+ *  Copyright (C) 2016-2022 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+
+#pragma once
```

Comment:
I think you got to the conclusion the name for this class is not the best one ever :)
For something that is designed with future headless in mind, GUI doesn't sound good. Maybe for now make this `ContentUtils` although a further split could possibly be done in the future?
For example I see some generic playback methods, methods 100% related to playlists, etc. But as a first step into decoupling this, clear +1 from me

Reply:
Yeah, I'm aware that `GUIUtils` isn't the best name.  I'd like to reach some sort of consensus on it though as renaming it all is a PITA.  The headless stuff only occurred to me after most of it was written, but it wasn't possible to get the functionality I wanted without starting to decouple stuff.
It's not 100% music as I had to decouple part of `CGUIMediaWindow::GetDirectory()` which is also used by video.